docs/builds/guides/integration/advanced-setup.md

```diff
@@ -130,7 +130,7 @@ module.exports = {
 				use: [ 'raw-loader' ]
 			},
 			{
-				// Or /ckeditor5-[^/]+\/theme\/[^/]+\.css$/ if you want to limit this loader
+				// Or /ckeditor5-[^/]+\/theme\/[\w-/]+\.css$/ if you want to limit this loader
```

Comment:
What does `\w-` mean?

Reply:
The `[\w-/]` bit translates to `[A-Za-z0-9_-/]`

Reply:
![image](https://user-images.githubusercontent.com/2270764/37278530-54330cb0-25e8-11e8-91cd-9fac3b30781a.png)

Eh. Thx.